app/views/govuk_publishing_components/components/docs/govspeak.yml

```diff
@@ -673,21 +673,9 @@ examples:
           <p>This is a summary</p>
         </div>
   buttons:
-    description: |
-      We're in process of migrating govspeak from using the old (toolkit) button styles to using the (govuk-frontend) button component from this gem. The example below demonstrates both kinds of buttons.
-
-      This documentation will be updated once that work is complete.
     data:
       block: |
         <p>
-          <a href="#" class="button" role="button">
-            Old button
-          </a>
-          <a href="#" class="button button-start" role="button">
-            Old start button
-          </a>
-        </p>
-        <p>
-          <a role="button" class="gem-c-button govuk-button" href="https://gov.uk/random">New button</a>
-          <a class="gem-c-button govuk-button govuk-button--start" role="button" href="https://gov.uk/random"> New start button <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" role="presentation" focusable="false"><path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z"></path></svg></a>
+          <a role="button" class="gem-c-button govuk-button" href="https://gov.uk/random">Button</a>
```

Comment:
Since `gem-c-button` extends `govuk-button` with no additional styles, are both classes needed?

Reply:
`gem-c-button` doesn't extend `govuk-button`, because that would duplicate `govuk-frontend` CSS inside the component gem CSS.

`gem-c-button` specifically provides a reduction in the bottom margin that `govuk-button` has: https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/stylesheets/govuk_publishing_components/components/_button.scss#L14

Reply:
Okay makes sense. Similar to the other comment, I take it the extra specificity gained by extending inside a `gem-c-govspeak` is needed because of overriding link styles?

https://github.com/alphagov/govuk_publishing_components/blob/fffb3678ad30dbaf50cf1bb85fef1bf2b01873a3/app/assets/stylesheets/govuk_publishing_components/components/govspeak/_button.scss#L5-L8

Reply:
Correct. It's not ideal, and I'm thinking about a better way of doing this, but that's outside the scope of this PR.